test/keymap.cpp

```diff
@@ -43,6 +43,50 @@ TEST_CASE("unset_key() and set_key()", "[KeyMap]")
 	}
 }
 
+TEST_CASE("unset_all_keys() clears the keymap from key bindings for a given context",
+	"[KeyMap]")
+{
+	SECTION("keymap has most of the keys set up by default") {
+		KeyMap k(KM_NEWSBOAT);
+
+		for (int i = OP_QUIT; i < OP_NB_MAX; ++i) {
+			if (i == OP_OPENALLUNREADINBROWSER ||
+					i == OP_MARKALLABOVEASREAD ||
+					i == OP_OPENALLUNREADINBROWSER_AND_MARK) {
+				continue;
+			}
+			REQUIRE(k.getkey(static_cast<Operation>(i), "all") != "<none>");
+		}
+	}
+
+	SECTION("\"all\" context clears the keymap from key bindings") {
```

Comment:
Maybe `…the _entire_ keymap…` to make it more shine out?

Reply:
How about `from _all_ key bindings`?

Reply:
Sounds good to me.